models/ratings.js

```diff
@@ -24,23 +24,35 @@ var ratings = {
           }
         ]
       },
-      function(result) {
+      function (result) {
         res.json(result);
       }
     );
   },
-  rateRecipe: function(req, res) {
+  rateRecipe: function (req, res) {
     orm.create(
       {
         table: "ratings",
         values: {
-          "rating_id": req.params.id,
+          "recipe_id": req.params.id,
```

Comment:
this should stay as "rating_id" because we're adding it from the recipe id in the api route params.

Reply:
It's editing the recipe_id column in the ratings table, rating_id is auto incremented

Reply:
There isn't a "recipe_id" column in ratings table

Reply:
we added it in after class yesterday, was part of the foreign key fix

Reply:
copy.